.github/ISSUE_TEMPLATE/recipe_template.md

```diff
@@ -1,8 +1,7 @@
 (fcb-recipeshorthand-template)=
 # Recipe Template
 
-+++
-<br/>
+
```

Comment:
Why was removing the separator and break lines necessary ?

Removing will impact the rendering (spacing between sections). If they don't break the new directive, it would be best to keep them.

Reply:
I think we should take them all out and add them as css. A markdown should not be used for styling in my opinion

Reply:
there is a lot of inconsistency with the horizontal rulers